src/foam/nanos/crunch/menus.jrl

```diff
@@ -12,6 +12,7 @@ p({
   "class": "foam.nanos.menu.Menu",
   "id": "crunch.crunchlab",
   "label": "Lab",
+  "keywords": ["crunchlab", "crunch lab", "debug crunch", "debug capabilities"],
```

Comment:
```suggestion
  "keywords": ["crunchlab", "crunch lab", "crunch", "lab", "debug crunch", "debug capabilities"],
```

Reply:
These keywords cover that case - typing `crunch` and `lab` alone brings up the menu item

Reply:
this is what `foam.mlang.predicate.Keyword` does
```javascript
          if ( s.toLowerCase().includes(arg) ) return true;
```